test/react-native/component.test.js

```diff
@@ -21,7 +21,7 @@ describe('App', () => {
     const query = gql`query people { allPeople(first: 1) { people { name } } }`;
     const data = { allPeople: { people: { name: 'Luke Skywalker' } } };
     const networkInterface = mockNetworkInterface({ request: { query }, result: { data } });
-    const client = new ApolloClient({ networkInterface });
+    const client = new ApolloClient({ networkInterface, addTypename: false });
```

Comment:
I thought we discussed making `mockNetworkInterface` support adding `__typename` to results automatically so we didn't have to worry about it in tests.